data/network/src/commonTest/kotlin/social/androiddev/common/network/MastodonApiTests.kt

```diff
@@ -74,14 +75,70 @@ class MastodonApiTests {
         assertNotNull(actual = result.getOrNull()?.uri)
     }
 
+    // TODO: fix loading json from resources
+    @Test
+    @Ignore
+    fun `create application request should fail with invalid response`() = runTest {
+        // TODO: Add test json
+        val content: String = ""
```

Comment:
since the response is 9 lines, you can add it inline instead
https://docs.joinmastodon.org/methods/apps/#create-an-application

Reply:
Cool sounds good to me. Wasn't sure yet where we wanted to go with these tests for API calls but I can just in-line the actual JSON like you suggested here and update these two tests I added.